app/router/student.py

```diff
@@ -458,6 +458,17 @@ async def verify_student(request: Request, student_id: str):
 
     logger.info(f"Verifying student: {student_id} with params: {query_params}")
 
+    # Check if this is EnableStudents auth group (temporary hardcoded check)
+    # TODO: Remove check for EnableStudents auth group when apaar_id merges with student_id
+    auth_group_id = query_params.get("auth_group_id")
+    is_enable_students = auth_group_id == "3"  # EnableStudents auth_group_id
+    if is_enable_students:
+        logger.info(
+            "Detected EnableStudents auth group - will try apaar_id fallback if needed"
+        )
```

Comment:
Will we keep adding more hardcoded group checks here as we expand the pilot to other groups? We should consider a more scalable approach, like a config or db driven check instead of hardcoding.